src/services/characterApi.js

```diff
@@ -1,5 +1,5 @@
 
-const URL = 'https://last-airbender-api.herokuapp.com/api/v1/characters/random?count=10';
+const URL = 'https://last-airbender-api.herokuapp.com/api/v1/characters/random?count=30';
 
 const CharacterApi = {
```

Comment:
Looks like you are missing some logic to actually query the API for specific allies and enemies of a character. Right now you just all return 30 random people that have no relation to the one you clicked on. 

`    getCharacters(query) {
        console.log('from api call', query);

        let url = URL;

        if(query) {
            url = `${URL}?${query}`;
        }


        return fetch(url)
            .then(response => response.json());
    }`

Reply:
You might need some logic like this in your API call

Reply:
You are passing this function a string when you use it in app but you don't do anything with it in this component.